python/lsst/cp/verify/mergeResults.py

```diff
@@ -128,11 +128,14 @@ def run(self, inputStats, camera, inputDims):
         outputStats = {}
         success = True
 
+        mergedStats = {}
```

Comment:
The goal of the merge is to allow there to be a hierarchical path from a single top level file (the output of `CpVerifyRunMergeTask`) that contains a list of all the failures, through the exposure level down to the detector/amp level.  The visualization notebooks (which I still need to extend for multi-detector cameras) can start with the top level file, and use that to determine what needs to be examined because of the failure.